api/src/pcapi/core/offers/exceptions.py

```diff
@@ -152,6 +152,11 @@ def __init__(self):  # type: ignore [no-untyped-def]
         )
 
 
+class EanFormatException(OfferCreationBaseException):
+    def __init__(self, error_message: str):  # type: ignore [no-untyped-def]
```

Comment:
Quand c'est possible, on essaie de ne pas ajouter de "# type: ignore", l'objectif étant de typer tout notre code, petit à petit. Ici, ça devrait être simple : 

```suggestion
    def __init__(self, error_message: str) -> None: 
```